runtime/near-vm-runner/src/tests/error_cases.rs

```diff
@@ -284,27 +318,21 @@ fn stack_overflow() -> Vec<u8> {
 
 #[test]
 fn test_stack_overflow() {
+    const COST: u64 = 63226248177;
```

Comment:
Why this one have a separated const COST, while other tests don't

Reply:
Fixed.